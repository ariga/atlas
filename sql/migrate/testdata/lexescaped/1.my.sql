create table t (c text default '\\');
create table t (c text default "\\");
create table t (c text default "\"");
create table t (c text default "\"" + '\'');
