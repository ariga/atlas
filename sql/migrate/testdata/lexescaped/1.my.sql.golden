create table t (c text default '\\');
-- end --
create table t (c text default "\\");
-- end --
create table t (c text default "\"");
-- end --
create table t (c text default "\"" + '\'');